server/filters/is-string.js

```diff
@@ -0,0 +1,3 @@
+export function isString(string) {
```

Comment:
nitpick – if w're checking whether or not it's a string, then the parameter name should probably be something less specific, like `input`, or `maybeString`